docs/backends/mssql.qmd

```diff
@@ -102,6 +120,20 @@ passing a properly-formatted MSSQL connection URL to `ibis.connect`:
 con = ibis.connect(f"mssql://{user}:{password}@{host}:{port}")
 ```
 
+### Using Integrated Authentication
+
+Integrated Authentication allows users to log in via NTLM or Kerberos.
+To make use of integrated authentication, use `ibis.mssql.connect` as documented
+above, but do not specify values for `user` or `password`. The PyODBC driver
+will then default to Integrated Authentication.
+
+Some environments may require you to set additional keywords to connect.
+Additional information is available at the following pages:
+
+<https://learn.microsoft.com/en-us/aspnet/web-api/overview/security/integrated-windows-authentication>
+
+<https://learn.microsoft.com/en-us/sql/connect/odbc/linux-mac/using-integrated-authentication>
```

Comment:
Do we want to format these links in any way (bulleted list? add title instead of raw link)? Or do these render fine as is? (I've never seen the `<{link}>` formatting used in qmd before, not does it do something different than just `{link}`? from rendering it looks the same to me).

Reply:
Changed to bulleted list.